library_models.js

```diff
@@ -0,0 +1,60 @@
+/*
+Exercise 5
+A JavaScript program that models a library, and should have a function that will add, remove and also list the properties in it.
+
+// For adding
+-Define an array of object.
+-Initialize a variable to hold the element and add it to array of object with push method.
+
+// For removing
+-Initialize another variable to hold one of the value in the object to remove with a filter method to return not equal to the value.
+
+// For listing
+-Define a callback function with the forEach method.
+-Finally print the callback.
+
+// For counting
+-Initialize a variable to hold a push method and finally print the variable.
+*/
+
+let library = [
+  {
+    title: "At The Heart Of Leadership",
+    author: "Joshua Freedman",
+    yearOfPublication: 2019,
+  },
+  {
+    title: "Forgive Yourself These Tiny Acts Of Self-Destruction",
+    author: "Jared Singer",
+    yearOfPublication: 2007,
+  },
+  {
+    title: "Being The Best Version Of You",
+    author: "Charlotte Moore",
+    yearOfPublication: 2010,
+  },
+  { title: "The Future", author: "Neil Hilborn", yearOfPublication: 2018 },
+];
+
+// For adding
+let bookAdded = {
+  title: "You Better Be Lighting",
+  author: "Andrea Gibson",
+  yearOfPublication: 2015,
+};
+library.push(bookAdded);
+console.log(library);
+
+// For removing
+const propertiesRemove = "The Future";
+const removeMethod = library.filter(
+  (properties) => properties.title !== propertiesRemove
+);
+console.log(removeMethod);
+
+// For counting
+let numberOfBooks = library.push();
+console.log("There are " + numberOfBooks + " books in the library");
+
+// For listing
+library.forEach((properties) => console.log(properties));
```

Comment:
@Dove-white this will display the array of books in the library. You are suppose to display each with their properties. Like this => **Title:** At The Heart Of Leadership **Author:** Joshua Freedman **Property Year:** 2019 for all books in the library.

Reply:
Thanks, boss. English hmmm 🤦‍♂️

Reply:
Please, the listing with properties is working. Look at the screenshot of it [Link](https://drive.google.com/file/d/1QlbXGvA_hV2l1CICkc1TNfelAIbDYFN9/view?usp=share_link)

Reply:
Yes. I have seen this result as well. In the screenshot, you are logging the objects in the array to the console instead of each book in the library with their title, author, and year of publication. 

### This is what the expected result is:
**Title**: At The Heart Of Leadership **Author:** Joshua Freedman Property Year: 2019
**Title**: Forgive Yourself These Tiny Acts Of Self-Destruction **Author:** Jared Singer Property Year: 2007
**Title**: Being The Best Version Of You **Author:** Charlotte Moore **Property Year:** 2010
**Title**: The Future **Author:** Neil Hilborn **Property Year:** 2018
**Title**: You Better Be Lighting **Author:** Andrea Gibson **Property Year:** 2015

### This is the question:
List all the books in the library: The program should list all the books in the library with their details (title, author, and year of publication).